influxql/ast.go

```diff
@@ -1406,6 +1407,7 @@ func (s *SelectStatement) validateAggregates(tr targetRequirement) error {
 	return nil
 }
 
+// HasDistinct is an exported method that checks if a select statment contains DISTINCT
```

Comment:
You can leave off the `exported method` part since anything starting with a capital letter is implicitly exported.

https://golang.org/ref/spec#Exported_identifiers